lib/internal/bootstrap/switches/is_main_thread.js

```diff
@@ -148,7 +148,11 @@ function getStdin() {
   switch (guessHandleType(fd)) {
     case 'TTY':
       const tty = require('tty');
-      stdin = new tty.ReadStream(fd);
+      stdin = new tty.ReadStream(fd, {
+        highWaterMark: 0,
```

Comment:
I think this is the problem. Should just remove the high watermark line.

IMO its also a bug in gauge but removing this should be sufficient.